cmd/eksctl-anywhere/cmd/generatehardware.go

```diff
@@ -130,7 +130,13 @@ func (hOpts *hardwareOptions) generateHardware(ctx context.Context) error {
 			return fmt.Errorf("failed reading csv: %v", err)
 		}
 
-		id := uuid.New().String()
+		var id string
```

Comment:
I had to update hardware to support optionally passing the hardware guid. This is required if you want to generate hardware.yaml for hardware that already exists in the tinkerbell stack which is required for the e2e test's since all the hardware already exists. Otherwise, the hardware generate command creates new guids for the hardware that does not match what is in our tinkerbell sandbox.